### PR TITLE
Fix total number in table for search results

### PR DESCRIPTION
**Pull request by wazsone**
# Contribution checklist

## Brief description
Total number is wrong for search results in table
image
image

## Checklist

* [ ] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: plugins/view-resources/src/components/Table.svelte [hunk 1]
```diff
@@ -230,7 +230,6 @@
     },
     {
       lookup,
-      limit: 1,
```
Comment (haiodo) [inline plugins/view-resources/src/components/Table.svelte:233]: Nope problem not with this. Also Total looks work fine in Table.

image

Comment (wazsone) [reply plugins/view-resources/src/components/Table.svelte:233]: @haiodo In qms after search Total shows 1, second screenshot. With this fix it works fine in qms. Do you know what else could affect this?

Comment (haiodo) [reply plugins/view-resources/src/components/Table.svelte:233]: It will request all data two times with this fix, please research why this is happening.

Comment (wazsone) [reply plugins/view-resources/src/components/Table.svelte:233]: You were right issue is not in the Table.

**Pull request closed without merging**